devops/scripts/create-sycl-user.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+if [[ $# -eq 0 ]]; then
+  # When launched without arguments, we assume that it was launched as part of
+  # CI workflow and therefore a different kind of user is created
```

Comment:
i wonder if we can check `GITHUB_ACTIONS`  here

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables

Reply:
Considering that all our containers are **only** for the purposes of CI and local development, I decided that they will all use `sycl_ci` by default. 

Those, who need `sycl` user are able to create it using that script manually as documented (8d931df2dd5af80b698a6f5a5155b0e5df7b4b51). Considering that `sycl` user won't be created by us anymore, I think that it is actually better to have this script react to an arguments instead of the environment (i.e. I want everything to be explicit)